src/main/scala/play/extras/geojson/LngLat.scala

```diff
@@ -0,0 +1,38 @@
+package play.extras.geojson
+
+import scala.collection.immutable.Seq
+import play.api.libs.json._
+import play.api.libs.functional._
+import play.api.libs.functional.syntax._
+import play.extras.geojson._
+
+/**
+ * LngLat is like LatLng except the ordering or latitude and longitude.
+ *
+ * In GeoJSON, and therefore Elasticsearch, the correct coordinate order 
+ * is longitude, latitude (X, Y) within coordinate arrays. This differs 
+ * from many Geospatial APIs (e.g., Google Maps) that generally use the 
+ * colloquial latitude, longitude (Y, X).
+ *
+ * @see http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/mapping-geo-shape-type.html
+ *
+ */
+case class LngLat(lng: Double, lat: Double)
+
+object LngLat {
+  implicit val lngLatFormat: Format[LngLat] = LngLatCrs.format
+  implicit val lngLatCrs: CrsFormat[LngLat] = LngLatCrs
+
+  def asLatLng(p: LngLat): LatLng = LatLng(p.lat, p.lng)
+  def fromLatLng(p: LatLng): LngLat = LngLat(p.lng, p.lat)
+
+}
+
+object LngLatCrs extends CrsFormat[LngLat] {
```

Comment:
This should have the `default` flag set to true, since it's still the default CRS, and so don't want it to output the `crs` property on every geojson object.